ADAL/src/ADAuthenticationErrorConverter.m

```diff
@@ -47,9 +49,10 @@ + (void)initialize
                              };
     
     s_errorCodeMapping = @{
-                           //sample format is like @"MSIDErrorDomain|-10000":@"-20000"
-                           @"MSIDErrorDomain|-51001":@"212",
-                           @"MSIDErrorDomain|-51301":@"101"
+                           MSIDErrorDomain:@{
+                                   NSNUMBER(MSIDErrorServerInvalidResponse) : NSNUMBER(AD_ERROR_SERVER_INVALID_RESPONSE),
+                                   NSNUMBER(MSIDErrorDeveloperAuthorityValidation) : NSNUMBER(AD_ERROR_DEVELOPER_AUTHORITY_VALIDATION)
+                                   }
```

Comment:
please also add additional MSIDErrors that we need to map

Reply:
sure, working on it :)